examples/_d2n-extended/flake.nix

```diff
@@ -50,12 +50,12 @@
         '')
       ];
       source = src;
-      settings = [
-        {
-          builder = "brp-new";
-          translator = "cargo-toml-new";
-        }
-      ];
+      projects.linemd = {
+        name = "linemd";
+        subsystem = "rust";
+        translator = "cargo-toml-new";
+        builder = "brp-new";
+      };
```

Comment:
This should point to `projects.toml` instead I think, with the translator and builder set correctly.